lib/messages.json

```diff
@@ -125,5 +125,6 @@
   "OPTION-VALUE-EXCEEDED": "option {option} exceeded limit of {value}{extra}",
   "INVALID-OPTION-COMBINATION": "option {option} can only be used with {rule}",
   "FILTER-FEATURE-NOT-SUPPORTED": "Filters do not support {feature} in this context.",
-  "HTTP-ERROR": "HTTP request failed with {status}: {message}"
+  "HTTP-ERROR": "HTTP request failed with {status}: {message}",
+  "CANNOT-FILTER-ON-TIME": "Cannot filter on \"time\". Use -to, -from, or -last instead."
```

Comment:
I'd make it clear that this only applies to `read`... i.e. "Cannot filter on \"time\" in read. Use -to, -from, or -last instead."